### PR TITLE
build(deps): Upgrade some libs

### PR DESCRIPTION
**Pull request by kissken**
set actual versions for some libs and fix dependabot alerts again

- github.com/aws/aws-sdk-go v1.43.31 => github.com/aws/aws-sdk-go v1.44.219
- github.com/blevesearch/bleve/v2 v2.3.2 => github.com/blevesearch/bleve/v2 v2.3.6
- github.com/dustin/go-humanize v1.0.0 => github.com/dustin/go-humanize v1.0.1
- github.com/pkg/errors v0.9.1 => removed
- github.com/rs/zerolog v1.20.0 => github.com/rs/zerolog v1.29.0
- github.com/smartystreets/assertions v1.2.0 => github.com/smartystreets/goconvey v1.7.2
- go.uber.org/automaxprocs v1.4.0 => removed

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,13 +5,13 @@ go 1.17
 require (
```
Comment (Tetrergeru) [inline go.mod:5]: Maybe we should put the name of upgraded libs into the PR name?

**Review by Tetrergeru — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (kissken) [reply go.mod:5]: i think in body PR will better, do u?

Comment (Tetrergeru) [reply go.mod:5]: Ok, that will do, I think

Comment (kissken) [reply go.mod:5]: done

**Pull request merged**